src/lib/drishti/CMakeLists.txt

```diff
@@ -343,16 +343,18 @@ endforeach()
 ###### PUBLIC API ##########
 ############################
 
-foreach(library ${DRISHTI_SDK_3RDPARTY_LIBS})
-  get_target_property(_TARGET_TYPE "${library}" TYPE)
-  if(NOT _TARGET_TYPE STREQUAL "INTERFACE_LIBRARY")
-    install(FILES $<TARGET_FILE:${library}> DESTINATION 3rdparty)
-  endif()
-endforeach()
+if(DRISHTI_INSTALL_3RDPARTY_LIBS)
```

Comment:
> You mean I can remove install(FILES $<TARGET_FILE:${library}> DESTINATION 3rdparty) logic instead of introducing DRISHTI_INSTALL_3RDPARTY_LIBS option?

Yes, this is no longer being used.